BuildAndRun.ps1

```
@@ -0,0 +1,241 @@
+# Extra args are MSBuild-style flags like /p:Platform=x64
+$extraArgs = $ExtraArgs
+
+# -- 0. Check Developer Mode --
+$devMode = $false
+try {
+    $regPath = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock"
+    if (Test-Path $regPath) {
+        $val = Get-ItemProperty $regPath -Name AllowDevelopmentWithoutDevLicense -ErrorAction SilentlyContinue
+        if ($val.AllowDevelopmentWithoutDevLicense -eq 1) { $devMode = $true }
+    }
+} catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)